### PR TITLE
Navigation

### PR DESCRIPTION
**Pull request by hessenh**
Navigation
Font
Moved some folders

### CODE REVIEW

File: src/components/navigation/navigation.jsx [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+class Navigation extends Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      loggedIn: false,
+    };
+  }
+  render() {
+    return (
+      <div className="bar">
```
Comment (johnolos) [inline src/components/navigation/navigation.jsx]: Possible to wrap everything in a nav-tag?

File: src/components/navigation/navigation.jsx [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+        <Link className="createCrawl" to="/create">Create a Crawl</Link>
+        <Link className="header" to="/">PubCrawler</Link>
+        <div className="auth">
+          {this.state.loggedIn ? <Link className="element" to="/logout">Logout</Link> :
+          <Link className="element" to="/login">Log in</Link>}
```
Comment (johnolos) [inline src/components/navigation/navigation.jsx]: Indent

File: src/components/navigation/navigation.jsx [hunk 3]
```diff
@@ -0,0 +1,27 @@
...
+    this.state = {
+      loggedIn: false,
+    };
+  }
+  render() {
+    return (
+      <div className="bar">
+        <Link className="createCrawl" to="/create">Create a Crawl</Link>
+        <Link className="header" to="/">PubCrawler</Link>
+        <div className="auth">
```
Comment (johnolos) [inline src/components/navigation/navigation.jsx]: Maybe take this out to a separate component?

Comment (hessenh) [reply src/components/navigation/navigation.jsx]: Hm.. Why? It is only links. I am currently changing "auth" to be "right" in a separate PR.

File: src/components/navigation/navigation.jsx (hunk 2 above)
Comment (hessenh) [reply src/components/navigation/navigation.jsx]: Tror den er riktig.

File: src/components/navigation/navigation.jsx (hunk 3 above)
Comment (johnolos) [reply src/components/navigation/navigation.jsx]: 👍

File: src/components/navigation/navigation.jsx (hunk 2 above)
Comment (johnolos) [reply src/components/navigation/navigation.jsx]: You can indent as well with eslint not freaking out.

In general I prefer 'single' code lines across multiple lines to be indented for readability. Your call.

**Review by johnolos — APPROVED** [review]

**Pull request merged**